src/content/en/fundamentals/instant-and-offline/get-started/textcontent-3.md

```diff
@@ -82,75 +72,45 @@ Looking at the bigger picture, the entire script block from which that snippet c
 HTML and CSS can be minified in the same way, so that you can achieve improved load times for 
 both formatting- and behavior-related code.
 
-Many online and desktop minification tools are available; here are some examples.
-
-### Minifier
+Many online and desktop minification tools are available; here's an example.
 
-This online tool minifies JavaScript or CSS via copy and paste. Paste the code, choose the 
+[Minifier](http://www.minifier.org/)
+is an online tool minifies JavaScript or CSS via copy and paste. Paste the code, choose the 
 type, and click `Minify`.
 
-[http://www.minifier.org/](http://www.minifier.org/)
-
 ![Minifier](images/image_300.png)
 
-### HTML Minifier
-
-Here's an online tool that also handles HTML, and automatically identifies the code type(s).
-
-[http://www.willpeavy.com/minifier/](http://www.willpeavy.com/minifier/)
-
-![HTML Minifier](images/image_301.png)
-
-### Kangax HTML Minifier
+Other minification tools include:
 
+- [HTML Minifier](http://www.willpeavy.com/minifier/):
+This online tool also handles HTML, and automatically identifies the code type(s).
+- [Kangax HTML Minifier](https://kangax.github.io/html-minifier/):
```

Comment:
I would recommend we choose a single HTML minification tool to recommend (Kangax one has been around a long time and is relatively trusted). 

The one thing to keep in mind when recommending third-party services for optimization is that we're not in control of what they do to the mangled content (always the minor risk of injecting something the user wasn't expecting). Picking ones that are well established does minimize this risk a little bit sometimes.